xdrip-get-entries.sh

```diff
@@ -713,20 +725,18 @@ function apply_lsr_calibration()
       remove_dexcom_bt_pair
       exit
     fi
+  fi
 
-    # Inner Calibrated BG boundary checks for case > 400
-    if [ $(bc <<< "$calibratedBG > 400") -eq 1 ]; then
-      log "Glucose $calibratedBG over 400 - setting noise level Heavy"
-      log "BG value will show in Nightscout but Openaps will not use it for looping"
-      noiseSend=4
-    fi
+  # Inner Calibrated BG boundary checks for case > 400
+  if [ $(bc <<< "$calibratedBG > 400") -eq 1 ]; then
+    log "Glucose $calibratedBG over 400; BG value of HI will show in Nightscout"
+    calibratedBG=401
+  fi
 
-    # Inner Calibrated BG boundary checks for case < 40
-    if [ $(bc <<< "$calibratedBG < 40") -eq 1 ]; then
-      log "Glucose $calibratedBG < 40 - setting noise level Light"
-      log "BG value will show in Nightscout and Openaps will conservatively use it for looping"
-      noiseSend=2
-    fi
+  # Inner Calibrated BG boundary checks for case < 40
+  if [ $(bc <<< "$calibratedBG < 40") -eq 1 ]; then
+    log "Glucose $calibratedBG < 40; BG value of LO will show in Nightscout"
+    calibratedBG=1
```

Comment:
no clue - we could ask in openaps gitter

Reply:
here's the scoop from Scott - looks like we need to change the 1 in my pr to 39: 

Scott Leibrand @scottleibrand 20:35
@tynbendad 39 and 401
below about 60 (depending on target), oref0 goes into LGS mode unless BG is rising faster than expected from insulin activity. so if the CGM is reading LOW (39), it will always zero temp.

Reply:
Setting noise is not required in this case as it will be handled by the normal noise calculation as you said above. This is good to go.